modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/SqlQueryProcessor.java

```diff
@@ -247,7 +261,8 @@ public synchronized void start() {
                 busyLock
         );
 
-        sqlSchemaManager.registerListener(prepareSvc);
+        //TODO IGNITE-17765 Do we need cache clear as cache already aware of catalog versioning.
```

Comment:
well, we should not refer in TODO to the ticket under which this TODO is being added